app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -66,7 +67,12 @@ DomainResponse handleOrder(DomainRequest request) {
 
         var demographics = patientDemographicsController.parseDemographics(request);
 
-        convertAndSendLabOrderUsecase.convertAndSend(demographics);
+        try {
+            convertAndSendLabOrderUsecase.convertAndSend(demographics);
+        } catch (UnableToSendLabOrderException e) {
```

Comment:
I'd write a new unit test for when the usecase throws an `UnableToSendLabOrderException` and assert that the `patientDemographicsController` was called in a certain way or something.

Reply:
lower t changed to T, nice catch!